Assets/Scripts/Framework/Utils/Libraries/LaneUtils.cs

```diff
@@ -0,0 +1,27 @@
+using ProefExamen.Framework.Gameplay.LaneSystem;
+
+namespace ProefExamen.Framework.Utils.Libraries.LaneUtils
+{
+    /// <summary>
+    /// A class holding LaneSystem calculations.
+    /// </summary>
+    public static class LaneUtils
+    {
+        /// <summary>
+        /// Returns a HitStatus enum that can be cast to an int for a score.
+        /// </summary>
+        /// <param name="differenceAlpha">A 0-1 alpha value that represents the accuracy of a note hit. 0 being getting the best score and 1 the lowest.</param>
+        /// <returns>The decided HitStatus based on the accuracy</returns>
+        public static HitStatus ReturnHitStatus(float differenceAlpha)
+        {
+            if (differenceAlpha > .75f)
+                return HitStatus.OK;
```

Comment:
```c#
if (differenceAlpha > .75f)
    return HitStatus.OK;
else if (differenceAlpha > .5f)
    return HitStatus.ALRIGHT;
else if (differenceAlpha > .25f)
    return HitStatus.NICE;
else
    return HitStatus.PERFECT;
```
Would be nice to make these "margin" values configurable in the inspector, incase we want to tweak the game.

Reply:
And also change the enums to UperCamelCase. See [code convention](https://github.com/lucasvdiepen/ProefExamen/wiki/Code-Conventions#enums).